src/CMakeLists.txt

```diff
@@ -21,29 +21,29 @@ if(NOT ${CMAKE_SYSTEM_NAME} STREQUAL "Emscripten"
     )
 endif()
 
-include(shared/sources.cmake)
-include(apps/vpn/cmake/sources.cmake)
-include(apps/vpn/cmake/sentry.cmake)
-
-target_link_libraries(mozillavpn PRIVATE shared-sources mozillavpn-sources glean lottie nebula translations)
-
 if(${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
-    set(MVPN_PLATFORM_NAME "linux")
+    set(MZ_PLATFORM_NAME "linux")
 elseif(${CMAKE_SYSTEM_NAME} STREQUAL "Windows")
-    set(MVPN_PLATFORM_NAME "windows")
+    set(MZ_PLATFORM_NAME "windows")
 elseif(${CMAKE_SYSTEM_NAME} STREQUAL "Darwin")
-    set(MVPN_PLATFORM_NAME "macos")
+    set(MZ_PLATFORM_NAME "macos")
 elseif(${CMAKE_SYSTEM_NAME} STREQUAL "Android")
-    set(MVPN_PLATFORM_NAME "android")
+    set(MZ_PLATFORM_NAME "android")
 elseif(${CMAKE_SYSTEM_NAME} STREQUAL "iOS")
-    set(MVPN_PLATFORM_NAME "ios")
+    set(MZ_PLATFORM_NAME "ios")
 elseif(${CMAKE_SYSTEM_NAME} STREQUAL "Emscripten")
-    set(MVPN_PLATFORM_NAME "wasm")
+    set(MZ_PLATFORM_NAME "wasm")
 endif()
 
-target_compile_definitions(mozillavpn PRIVATE "$<$<CONFIG:Debug>:MVPN_DEBUG>")
-target_compile_definitions(mozillavpn PRIVATE "MVPN_$<UPPER_CASE:${MVPN_PLATFORM_NAME}>")
-include(apps/vpn/cmake/${MVPN_PLATFORM_NAME}.cmake)
+include(shared/sources.cmake)
+include(apps/vpn/cmake/sources.cmake)
+include(apps/vpn/cmake/sentry.cmake)
+
+target_link_libraries(mozillavpn PRIVATE shared-sources mozillavpn-sources glean lottie nebula translations)
+
+target_compile_definitions(mozillavpn PRIVATE "$<$<CONFIG:Debug>:MZ_DEBUG>")
```

Comment:
what is this mozillavpn? (apologies if this should be obvious i'm behind on reviewing this series of prs)

Reply:
This is the "mozillaVPN" target, the client.

Reply:
ok. i think i'm going to need an overview of how this will work...will there be lots of target_compile_definitions at this level? ... like one for the tests etc.